phlib/include/banned.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+//
+// These functions are banned in process hacker and associated plug-ins.
+//
+
+__declspec(deprecated("LoadLibraryA is banned, use PsLoadLibrarySafe instead (see: banned.h)."))
```

Comment:
Since C++14 there is `[[deprecated]]` attribute officially supported in the language.
https://en.cppreference.com/w/cpp/language/attributes/deprecated

Reply:
C++ standards 😱